packages/components/src/local/route/helpers/planned-routes-for.ts

```diff
@@ -0,0 +1,56 @@
+import { LatLng } from 'leaflet'
+
+import SergeHex from '../../mapping/types/serge-hex'
+import SergeGrid from '../../mapping/types/serge-grid'
+import hexNamed from '../../hex-grid/helpers/hex-named'
+
+export interface RouteData {
```

Comment:
This should probably be in a types folder.

Reply:
Done.